### DISCUSSION TITLE
Change the default mode

### DISCUSSION THREAD

**Discussion by pablorivas-robledo** [AllanChain/logseq-live-math#32] [Q&A] [answered]
Hello, I have tried to configure the plugin so that the the default mode is not Mathematics, but rather Latex.

![image](https://github.com/user-attachments/assets/899ef86a-b205-47c1-bf62-800006692cee)

Is there a say to do it? At the moment I need to change it every time I enter math mode.

**Comment by AllanChain** [comment, marked as answer]
It's currently not possible to change the default mode. However, you can quickly switch to LaTeX mode with <kbd>Esc</kbd> key.

**Comment by pablorivas-robledo** [comment]
Ah, no worries. The Esc key trick works fine for me.
Thanks.
